symfony/plugins/orangehrmPimPlugin/config/routes.yaml

```diff
@@ -333,6 +333,52 @@ apiv2_pim_optional_field:
     id: 0
     _api: OrangeHRM\Pim\Api\OptionalFieldAPI
 
+apiv2_pim_reporto_supervisors:
+  path: /api/v2/pim/employees/{empNumber}/supervisors
+  controller: OrangeHRM\Core\Controller\Rest\V2\GenericRestController::handle
+  methods: [ GET, POST, DELETE ]
+  defaults:
+    _api: OrangeHRM\Pim\Api\EmployeeSupervisorAPI
+  requirements:
+    empNumber: '\d+'
+
+apiv2_pim_reporto_supervisor:
+  path: /api/v2/pim/employees/{empNumber}/supervisors/{id}
+  controller: OrangeHRM\Core\Controller\Rest\V2\GenericRestController::handle
+  methods: [ GET, PUT ]
+  defaults:
+    _api: OrangeHRM\Pim\Api\EmployeeSupervisorAPI
+  requirements:
+    empNumber: '\d+'
+    id: '\d+'
+
+apiv2_pim_reporto_subordinate:
+  path: /api/v2/pim/employees/{empNumber}/subordinates/{id}
+  controller: OrangeHRM\Core\Controller\Rest\V2\GenericRestController::handle
+  methods: [ GET, PUT ]
+  defaults:
+    _api: OrangeHRM\Pim\Api\EmployeeSubordinateAPI
+  requirements:
+    empNumber: '\d+'
+
+apiv2_pim_reporto_subordinates:
+  path: /api/v2/pim/employees/{empNumber}/subordinates
+  controller: OrangeHRM\Core\Controller\Rest\V2\GenericRestController::handle
+  methods: [ GET, POST, DELETE ]
+  defaults:
+    _api: OrangeHRM\Pim\Api\EmployeeSubordinateAPI
+  requirements:
+    empNumber: '\d+'
+
+apiv2_pim_reporto_allowed_employees:
+  path: /api/v2/pim/employees/{empNumber}/reportTo/allowed
```

Comment:
`/api/v2/pim/employees/{empNumber}/report-to/allowed`
kebab-case prefered for api paths